reloj-checador/db/schema.rb

```diff
@@ -15,7 +15,6 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-
```

Comment:
It doesn't look like we are creating a migration at all.... perhaps... did you miss to add it?

Reply:
It's in another branch

Reply:
Then, are these changes needed in this branch? 🤔

Reply:
Can we just remove the commit? 😄